src/scss/widget.scss

```diff
@@ -0,0 +1,396 @@
+/*!
```

Comment:
Ik gebruik zelf geen scss dus neem dit met een korrel zout, maar dit lijkt mij heel erg veel css voor het weergeven van een tabelletje, en daarbij, je kunt nu toch variabelen gebruiken, en toch zijn alle kleuren in gedefinieert in hex waarden. Ik kan alleen maar hopen dat dit nog geoptimaliseerd wordt ergens.